db/migrations/20190917093141_interrupt_layout_migration.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Copyright (C) 2019 Xibo Signage Ltd
+ *
+ * Xibo - Digital Signage - http://www.xibo.org.uk
+ *
+ * This file is part of Xibo.
+ *
+ * Xibo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * Xibo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Xibo.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+use Phinx\Migration\AbstractMigration;
+
+class InterruptLayoutMigration extends AbstractMigration
+{
+    /** @inheritdoc */
+    public function change()
+    {
+        $scheduleTable = $this->table('schedule');
+
+        // Add a two new columns to Display table, newCmsAddress and newCmsKey
+        if (!$scheduleTable->hasColumn('shareOfVoice')) {
+            $scheduleTable
+                ->addColumn('shareOfVoice', 'integer', ['limit' => 100, 'default' => null, 'null' => true])
```

Comment:
that is a large integer! i think a TINY_INT would do the trick? also the comment might want updating ;)